static/app/views/organizationStats/usageStatsPerMin.tsx

```diff
@@ -12,11 +12,9 @@ type Props = {
   dataCategory: DataCategoryInfo['plural'];
   organization: Organization;
   projectIds: number[];
-} & DeprecatedAsyncComponent['props'];
+};
 
-type State = {
-  orgStats: UsageSeries | undefined;
-} & DeprecatedAsyncComponent['state'];
+type UsageStatsType = UsageSeries | undefined;
```

Comment:
it seems weird that the api would ever be undefined, i think the old type was likely wrong but attempting to cover AsyncComponent's state of null when the api returns 400.